esphome/components/uptime/text_sensor/__init__.py

```diff
@@ -1,19 +1,54 @@
 import esphome.codegen as cg
-from esphome.components import text_sensor
+from esphome.components import text_sensor, time
 import esphome.config_validation as cv
-from esphome.const import ENTITY_CATEGORY_DIAGNOSTIC, ICON_TIMER
+from esphome.const import (
+    CONF_DURATION,
+    CONF_TIME_ID,
+    ENTITY_CATEGORY_DIAGNOSTIC,
+    ICON_TIMER,
+)
+
+CONF_TIMESTAMP = "timestamp"
```

Comment:
```suggestion
TYPE_TIMESTAMP = "timestamp"
```
It's a value, not a key name. Same for CONF_DURATION.